modules/users/server/routes/admin.server.routes.js

```diff
@@ -4,22 +4,28 @@
  * Module dependencies
  */
 var adminPolicy = require('../policies/admin.server.policy'),
-  admin = require('../controllers/admin.server.controller');
+  admin = require('../controllers/admin.server.controller'),
+  passport = require('passport'),
+  express = require('express');
 
 module.exports = function (app) {
-  // User route registration first. Ref: #713
-  require('./users.server.routes.js')(app);
+  var router = express.Router();
 
-  // Users collection routes
-  app.route('/api/users')
+  //Set JWT Auth for all Admin Routes
+  router.all('*', passport.authenticate('jwt', { session: false }));
+
+  router.route('/')
     .get(adminPolicy.isAllowed, admin.list);
 
   // Single user routes
-  app.route('/api/users/:userId')
+  router.route('/:userId')
     .get(adminPolicy.isAllowed, admin.read)
     .put(adminPolicy.isAllowed, admin.update)
     .delete(adminPolicy.isAllowed, admin.delete);
 
   // Finish by binding the user middleware
-  app.param('userId', admin.userByID);
+  router.param('userId', admin.userByID);
+
+  app.use('/api/admin/users', router);
```

Comment:
Here is another use case.  People have been asking about logging.  With this implementation you can log all admin actions easily by doing router.all('*', myloggingfunction); in the user admin router. This would not be available if it was all on the same router.  Separating them onto their own router makes a lot of additional features possible.  Adding an /admin route in my mind is worth it.